hypervisor/arch/x86/guest/vcpu.c

```diff
@@ -263,6 +263,51 @@ void set_ap_entry(struct vcpu *vcpu, uint64_t entry)
 	vcpu_set_rip(vcpu, 0UL);
 }
 
+
+/* NOTE: set_bsp_rm_entry & set_bsp_protect_regs is only temporary function
+ * called by UOS. Once we make UOS to use hypercall to set BSP entry,
+ * we will remove these two functions.
+ */
+
+/*
+ * @pre reset_vcpu_regs is called in advance
+ */
+void set_bsp_rm_entry(struct vcpu *vcpu)
```

Comment:
use realmode rather than rm?

Reply:
OK. I will scan all patchset and update rm -> realmode